proto/supernode/storage_challenges.proto

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2021-2021 The Pastel Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+syntax = "proto3";
+
+option go_package = "github.com/pastelnetwork/gonode/proto/supernode;supernode";
+
+package supernode;
+
+service StorageChallenge {
+  rpc GenerateStorageChallenges(GenerateStorageChallengesRequest) returns (GenerateStorageChallengesReply);
+  rpc ProcessStorageChallenge(ProcessStorageChallengeRequest) returns (ProcessStorageChallengeReply);
+  rpc VerifyStorageChallenge(VerifyStorageChallengeRequest) returns (VerifyStorageChallengeReply);
+}
+
+message StorageChallengeData {
+  string message_id = 1;
+  enum messageType {
```

Comment:
why not define these enums outside of StorageChallengeData, and make it more compact, like:
```
enum MessageType {
    UNKNOWN = 0;
    STORAGE_CHALLENGE_ISSUANCE_MESSAGE = 1;
...
}
```
so we can have generated code:
```
type MessageType int32

const (
	MessageType_UNKNOWN                            MessageType = 0
	MessageType_STORAGE_CHALLENGE_ISSUANCE_MESSAGE MessageType = 1
)

```